### ISSUE TITLE
H2 References

### ISSUE THREAD

**Issue by adrianevs** [dwp-forge/refnotes#31]
How can I make when exist a note in a page to be automated an h2 for "References" , before the long line, before list start ?

Thanks

**Comment by dwp-forge** [comment]
Sorry, cannot I understand what you are trying to achieve. Could you rephrase the question?

**Comment by adrianevs** [comment]
Hy, Sorry I will try to make it more clear.

I suggest to make that plugin to add an "H2" with the text "Refernces" in the buttom o page above the links are dispayed. And to make it automatic when exist references on page.

Please take a look in the prt scrn.
![screen shot 2016-03-31 at 12 11 38 am copy](https://cloud.githubusercontent.com/assets/18171505/14157838/91f6dd9a-f6d5-11e5-83bb-3959a5b01399.png)

**Comment by dwp-forge** [comment]
Could be done, but this is going to simplify only one (though, likely, most common) use case. On the other hand, the implementation of such feature would need to take into account other use cases, so there would be required a number of configuration settings (e.g. on/off switch, heading level, localization, etc.)

Also the plugin was designed as a drop-in replacement of DokuWiki built-in footnotes. Its default behavior (using minimal syntax) tries to emulate the footnotes as close as possible. So enabling the heading would require extra syntax anyway and there would be not much to gain.

Given these pros and cons I don't think adding such a feature would be worthwhile.
